source/opcode.h

```diff
@@ -31,166 +31,84 @@
 #include "libspirv/libspirv.h"
 #include "table.h"
 
-// Functions
-
-/// @brief Get the name of the SPIR-V generator.
-///
-/// See the registry at
-/// https://www.khronos.org/registry/spir-v/api/spir-v.xml
-///
-/// @param[in] generator Khronos SPIR-V generator ID
-///
-/// @return string name
+// Gets the name of the SPIR-V generator.
```

Comment:
Returns the name of a registered SPIR-V generator as a null-terminated string.  If the generator is not known, then returns the string "Unknown".  The generator parameter should be most significant 16-bits of the generator word in the SPIR-V module header.  Values are registered at the SPIR-V XML registry: https://www.khronos.org/registry/spir-v/api/spir-v.xml